projects/dsp-ui/src/lib/viewer/operations/add-value/add-value.component.scss

```diff
@@ -0,0 +1,60 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: 70% 30%;
+  grid-template-rows: auto auto;
+}
+
+.value-component,
+.crud-buttons{
+  vertical-align: top;
+}
+
+.value-component {
+  grid-row-start: 1;
+  grid-row-end: auto;
+  grid-column-start: 1;
+  grid-column-end: 2;
+}
+
+.crud-buttons {
+  grid-row-start: 1;
+  grid-row-end: auto;
+  grid-column-start: 2;
+  grid-column-end: 3;
+  padding-top: 0.5em;
+}
+
+:host ::ng-deep .child-value-component,
```

Comment:
@kilchenmann told me it is possible to avoid the use of ng-deep by moving this css overwrite to the root style file (lib/assets/style/*) - a new file _viewer.scss can be created in this directory for the viewer module e.g. _search.scss

Reply:
fixed in 0e070bd, there are a couple ::ng-deep selectors still in some scss files, but I think itäs out of scope for this PR